### PR TITLE
Ifu 4 26 2023

### PR DESCRIPTION
**Pull request by micmelesse**
https://github.com/ROCmSoftwarePlatform/frameworks-internal/issues/4310

### CODE REVIEW

File: .github/workflows/amd-offline-tests.yml [hunk 1]
```diff
@@ -33,8 +33,8 @@ jobs:
           apt remove -y cmake
           apt update
           apt install -y git
-          python3 -m pip install --upgrade pre-commit
-          python3 -m pre_commit run --all-files
+          # python3 -m pip install --upgrade pre-commit
```
Comment (jayfurmanek) [inline .github/workflows/amd-offline-tests.yml]: Do we expect to re-enable this at some point? I think it may be better to just remove them.

Comment (jayfurmanek) [reply .github/workflows/amd-offline-tests.yml]: n/m, its removed later

**Comment by jayfurmanek** [comment]
Thanks @micmelesse !
Can you rebase to squash your added commits at the bottom down to a fewer with more descriptive labels?

Maybe into:
 - fix libdevice
 - fix lit tests
 - fix workflows
 - fix exp and constexpr tests

**Comment by micmelesse** [comment]
@jayfurmanek Done

**Review by jayfurmanek — APPROVED** [review]

**Pull request merged**